packages/sysutils/busybox/scripts/init

```diff
@@ -29,6 +29,9 @@
   /bin/busybox mkdir -p /sysroot
   /bin/busybox mkdir -p /storage
 
+  # Just in case... problems with ssh if not correctly owned
+  chown 0:0 /storage 2>/dev/null
```

Comment:
see https://github.com/MilhouseVH/LibreELEC.tv/blob/6bfc64bc24dfb3a694df6ab13d103aad624976df/packages/sysutils/busybox/scripts/init#L1021-L1024

there ia a reason for using /bin/busybox there. some applets (like echo) are special - part of ash, so they'll run fine after /proc is not yet there or gone, but most wont..